### ISSUE TITLE
terraform dry run failing after creating graylog alarm callback

### ISSUE THREAD

**Issue by daryl-d** [suzuki-shunsuke/go-graylog#139]
Hi @suzuki-shunsuke,

Have been trying out graylog alarm callback with custom third party api as specified [here](https://github.com/suzuki-shunsuke/go-graylog/blob/master/terraform/docs/alarm_callback.md#type-other-third-partys-alarm-callback).

Creating the resource itself works perfectly.

```hcl
resource "graylog_alarm_callback" "pagerduty" {
  type = "org.graylog2.alarmcallbacks.pagerduty.PagerDutyAlarmCallback"
  stream_id = "${data.graylog_stream.default_stream.id}"
  title = "Test"

  general_string_configuration = {
    service_key = "<integration key>"
    incident_key_prefix = "Graylog/"
    client = "Graylog"
    client_url = "<graylog url>"
  }

  general_bool_configuration = {
    use_custom_incident_key = "true"
  }
}
```

However, whenever I try to rerun the terraform, the dry run always fails

```
Error: Error refreshing state: 1 error occurred:
--
  | * graylog_alarm_callback.pagerduty: 1 error occurred:
  | * graylog_alarm_callback.pagerduty: graylog_alarm_callback.pagerduty: general_bool_configuration.use_custom_incident_key: can only set full map

```

I've traced this down to the actual terraform source code [here](https://github.com/hashicorp/terraform/blob/master/helper/schema/field_writer_map.go#L78)

Environment:
graylog provider version: `6.1.0`
terraform version: `0.11.14`
graylog version: `3.0.2+1686930`

graylog provider defintion

```hcl
provider "graylog" {
  web_endpoint_uri = "<url>"
  api_version      = "v3"
  auth_name        = "<user name>"
  auth_password    = "<password>"
}
```

Please let me know if you need any more info.

I really like what your are doing 👍

**Comment by suzuki-shunsuke** [comment]
Hi @daryl-d 
Thank you for your feedback.
I have reproduced it.
I will check it.

Thank you.

**Comment by suzuki-shunsuke** [comment]
* https://github.com/hashicorp/terraform/blob/v0.11.14/helper/schema/field_writer_map.go#L67-L80
* https://github.com/hashicorp/terraform/blob/v0.12.5/helper/schema/field_writer_map.go#L67-L80
* https://github.com/suzuki-shunsuke/go-graylog/blob/v6.1.0/terraform/graylog/resource_alarm_callback.go#L389

**Comment by suzuki-shunsuke** [comment]
I have created the pull request https://github.com/suzuki-shunsuke/go-graylog/pull/140.

**Comment by suzuki-shunsuke** [comment]
I have released [v6.1.1](https://github.com/suzuki-shunsuke/go-graylog/releases/tag/v6.1.1) .

**Comment by daryl-d** [comment]
works perfectly @suzuki-shunsuke :)

thanks heaps for this
